main/networking/Ansible.js

```diff
@@ -6,86 +6,76 @@ import _ from 'lodash';
 
 import RendererBridge from '../RendererBridge';
 import { updateConsole } from '../../renderer/actions/ConsoleActions';
-import {
-  ansibleDisconnect,
-  infoPerMessage,
-  updateCodeStatus,
-} from '../../renderer/actions/InfoActions';
+import { ansibleDisconnect } from '../../renderer/actions/InfoActions';
 import { updatePeripherals } from '../../renderer/actions/PeripheralActions';
-import { robotState, Logger, defaults } from '../../renderer/utils/utils';
-import FCObject from './FieldControl';
-
-const DawnData = (new protobuf.Root()).loadSync(`${__dirname}/ansible.proto`, { keepCase: true }).lookupType('DawnData');
-const { StudentCodeStatus } = DawnData;
-
-const RuntimeData = (new protobuf.Root()).loadSync(`${__dirname}/runtime.proto`, { keepCase: true }).lookupType('RuntimeData');
-const Notification = (new protobuf.Root()).loadSync(`${__dirname}/notification.proto`, { keepCase: true }).lookupType('Notification');
+import { Logger, defaults } from '../../renderer/utils/utils';
+
+/**
+ * UDP Recv (Runtime -> Dawn)
+ * Device Data Array (sensors), device.proto
+ */
+const RecvDeviceProto = (new protobuf.Root()).loadSync('protos/device.proto', { keepCase: true }).lookupType('DevData');
+
+/**
+ * UDP Send (Dawn -> Runtime)
+ * Gamepad Data, gamepad.proto
+ */
+const SendGamepadProto = (new protobuf.Root()).loadSync('protos/gamepad.proto', { keepCase: true }).lookupType('GpState');
+
+/**
+ * TCP Recv (Runtime -> Dawn)
+ * Challenge Data (output values), text.proto
+ * Log Data (console), text.proto
+ */
+const RecvChallengeProto = (new protobuf.Root()).loadSync('protos/text.proto', { keepCase: true }).lookupType('Text');
+const { RecvLogProto } = RecvChallengeProto;
+
+/**
+ * TCP Send (Dawn -> Runtime)
+ * Run Mode Data (TELEOP, ESTOP), run_mode.proto
+ * Device Data Array (preference filter), device.proto
```

Comment:
Another small detail—before the feature is implemented, you don't need to worry about sending this packet at all; Runtime will take care of it and send all parameters for all devices by default before the feature is implemented.